Computer History Museum Archive.js

```diff
@@ -0,0 +1,1291 @@
+						"tag": "Silicon Graphics"
+					},
+					{
+						"tag": "Software"
+					},
+					{
+						"tag": "Venture Capital"
+					},
+					{
+						"tag": "Video Art"
+					},
+					{
+						"tag": "personal computers (PCs)"
+					},
+					{
+						"tag": "software industry"
+					}
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102645840",
+		"items": [
+			{
+				"itemType": "videoRecording",
+				"title": "Challenges and Directions in Fault-Tolerant Computing, lecture by Jack Goldberg",
+				"creators": [
+					{
+						"firstName": "Jack",
+						"lastName": "Goldberg",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "1985-10-02",
+				"abstractNote": "Label taped to the video case reads: \"Two decaes of theoritical and experimental work and numerous recent successful applications have established fault tolerance as a standard objective of high speed, satisfaction of fault tolerance requirements cannot be demonstrated by testing alone, but requires formal analysis.  Most of the work in fault tolerance has been concerned with developing effective design techniques.  Recent work on reliabilitymodeling and formal proof of fault tolerant design and implementation is laying a foundation for a more rigorous design discipline.  The scope of concern has also expanded to include any source of computer reliability, such as design mistakes, in software, hardware, or at any system level.  Current art is barely able to keep up with the rapid pace of computer technology, the stresses of new applications and the new expansion in scope of concerns.  Particular challenges lie in coping with the imperfections of the ultrasamll, i.e., high density VLSI, and the ultra-large, i.e., large software systems.  It is clear that fault tolerance cannot be \"added\" to a design and must be integrated with other design objectives.  Simultaneous demands in future systems for high performance, high security, high evolvability and high fault tolerance will require new theoretical models of computer systems and a much closer integration of practical design techniques.  The talk will discuss the widening scope of research into computer dependability.  New issues include tolerance of design errors (including software), operator errors, and the safety of computer-controlled systems.\"",
+				"archive": "Computer History Museum",
+				"extra": "Catalog Number: 102645840\nPlatform: NTSC VHS VCR\nCategory: Lecture\nSeries Title: Stanford Computer Forum Distinguished Lecture Series",
+				"libraryCatalog": "Computer History Museum Archive",
+				"place": "Palo Alto, CA, US",
+				"studio": "Stanford University. Stanford Computer Forum",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102706809",
+		"items": [
+			{
+				"itemType": "book",
```

Comment:
This is `Type: Audio` on the page, and in Zotero should be an `audioRecording` item, which has `format` field for the `Standard audio cassette` value.

Reply:
Thanks! Fixed. BTW, Scafford doesn't have the format for the audio- & video-recording formats fields. Perhaps needs an update?

Reply:
It does on my end, at least. The format field for `videoRecording` is called `videoRecordingFormat`, and for `audioRecording` it's called `audioRecordingFormat`.

Reply:
> It does on my end, at least. The format field for `videoRecording` is called `videoRecordingFormat`, and for `audioRecording` it's called `audioRecordingFormat`.

Oh I see! Thanks! Fixed. Is it ready to merge?

Reply:
I still see the two `replace('https', 'http')` lines, but after that's gone, yes!

Reply:
ahh, sorry about that! The earlier change was made directly on Github and I forgot to pull. Just updated! Thanks so much for all the helps.